yaml/pois.yaml

```diff
@@ -1697,6 +1704,21 @@ filters:
     output:
       <<: *output_properties
       kind: quarry
+  - filter:
+      leisure:
+        - adult_gaming_centre
+        # fairly common misspelling
+        - adult_gaming_center
+    min_zoom: 17
+    output:
+      <<: *output_properties
+      kind: gaming
```

Comment:
I think the kind should be `adult_gaming_centre`

Reply:
Changed in 96110df.